src/main/kotlin/net/ccbluex/liquidbounce/features/module/modules/exploit/disabler/disablers/DisablerBlocksMCVelocity.kt

```diff
@@ -0,0 +1,42 @@
+/*
+ * This file is part of LiquidBounce (https://github.com/CCBlueX/LiquidBounce)
+ *
+ * Copyright (c) 2015 - 2024 CCBlueX
+ *
+ * LiquidBounce is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * LiquidBounce is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with LiquidBounce. If not, see <https://www.gnu.org/licenses/>.
+ */
+package net.ccbluex.liquidbounce.features.module.modules.exploit.disabler.disablers
+
+import net.ccbluex.liquidbounce.config.ToggleableConfigurable
+import net.ccbluex.liquidbounce.event.repeatable
+import net.ccbluex.liquidbounce.features.module.modules.exploit.disabler.ModuleDisabler
+import net.minecraft.network.packet.c2s.play.ClientCommandC2SPacket
+
+object DisablerBlocksMCVelocity : ToggleableConfigurable(
+    ModuleDisabler,
+    "BlocksMCVelocity",
+    false
+) {
+
+    val repeatable = repeatable {
+        when (player.hurtTime) {
+            9 -> {
+                // two in the same tick so you dont flag movement
+                network.sendPacket(ClientCommandC2SPacket(player, ClientCommandC2SPacket.Mode.PRESS_SHIFT_KEY))
+                network.sendPacket(ClientCommandC2SPacket(player, ClientCommandC2SPacket.Mode.RELEASE_SHIFT_KEY))
+            }
+        }

```

Comment:
Not a disabler.

Reply:
it disables the verus velocity checks so you can use whatever velocity you want ?